.github/workflows/release_shim.yml

```
@@ -0,0 +1,36 @@
+  workflow_dispatch:
+
+jobs:
+  release-shim:
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write        # PyPI trusted publishing(OIDC), 与主包一致
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Set up Python 3.11
+        uses: actions/setup-python@v4
```

actionlint (1.7.12)

[error] the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)